src/pages/workspace/workflows/ToggleSettingsOptionRow.tsx

```diff
@@ -0,0 +1,92 @@
+import React, {useState} from 'react';
+import {View} from 'react-native';
+import type {SvgProps} from 'react-native-svg';
+import Icon from '@components/Icon';
+import Switch from '@components/Switch';
+import Text from '@components/Text';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import type {Styles} from '@styles/index';
+
+type ToggleSettingOptionRowProps = {
+    icon: React.FC<SvgProps>;
+    title: string;
+    subtitle: string;
+    hasBeenToggled: boolean;
+    onToggle: (isEnabled: boolean) => void;
+    subMenuItems?: React.ReactNode;
+};
+
+function ToggleSettingOptionRow({icon, title, subtitle, onToggle, subMenuItems, hasBeenToggled}: ToggleSettingOptionRowProps) {
+    const [isEnabled, setIsEnabled] = useState(hasBeenToggled);
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const ICON_SIZE = 48;
+    const toggleSwitch = () => {
+        setIsEnabled(!isEnabled);
+        onToggle(!isEnabled);
+    };
+
+    // Since these styles are only used in this component we define them here
```

Comment:
I think it's best to styles.ts to separate concerns

Reply:
> I think it's best to styles.ts to separate concerns

The thing is these styles are one-off styles and will not be repeated elsewhere. We previously discussed it with @ishpaul777

Reply:
[Alright I've reused app styles ](https://github.com/Expensify/App/pull/36345/commits/8b5e067d1f41535ebb093e0832d39ee22973bd81). 

I think that's what you wanted @ishpaul777?